src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -167,50 +153,49 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
         if (distributeToRecipients) {
             String optionFragment =
                     Templates.populateTemplate(optionFragmentTemplate,
-                            "${qnIdx}", Integer.toString(qnIdx),
-                            "${responseIdx}", Integer.toString(responseIdx),
-                            "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled",
-                            "${constSumOptionVisibility}", "style=\"display:none\"",
-                            "${constSumOptionPoint}", existingConstSumResponse.getAnswerString(),
-                            "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}", Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
-                            "${constSumOptionValue}", "");
+                            Templates.FeedbackQuestionDetailsConsts.QUESTION_INDEX, Integer.toString(qnIdx),
```

Comment:
If you do that, then the constants should start with a special prefix. e.g. `SLOT_`
Otherwise it will be hard to distinguish these constants from others.